api/python/cell_census/src/cell_census/open.py

```diff
@@ -31,9 +31,34 @@ def _open_soma(description: CensusReleaseDescription) -> soma.Collection:
 def open_soma(*, census_version: Optional[str] = "latest", uri: Optional[str] = None) -> soma.Collection:
     """
     Open the Cell Census by version (name) or URI, returning a soma.Collection containing
-    the top-level census.
+    the top-level census.  Raises error if census_versio is specified and unknown, or if
+    neither ``uri`` or ``census_version`` are specified.
 
     TODO: add platform_config hook when it is further defined, allowing config overrides.
+
+    Parameters
+    ----------
+    census_version : Optional[str]
+        The tag/name of the Census, e.g., "latest"
+    uri : Optional[str]
+        The URI containing the Census SOMA objects. If specified, will take precedence
+        over ``census_version`` parameter.
```

Comment:
well...that is a question of what the underlying storage engine supports, and given that SOMA is multi-engined, it could vary. That said, tiledbsoma does (primarily) s3 and file.  Probably best to show in the examples section, rather than the parameters, given that this could vary over time.